atomic_reactor/util.py

```diff
@@ -649,18 +649,22 @@ def default(self):
         return self.v2 or self.v1
 
 
-def get_manifest_digests(image, registry, insecure=False, dockercfg_path=None,
-                         versions=('v1', 'v2')):
+def get_media_type(version):
+    return 'application/vnd.docker.distribution.manifest.{}+json'.format(version)
```

Comment:
Is this module-level function name descriptive enough? Wondered if it should be `get_manifest_media_type`.